### DISCUSSION TITLE
No update After update skin

### DISCUSSION THREAD

**Discussion by fvirgola80** [poblabs/weewx-belchertown#784] [Q&A] [answered]
hello, yesterday I did the update of the skin version 1.3 b2. I did the clean install again and re-entered all the changes. I realized that when I open the website 
www.meteocivitavecchia.it
 I have the update on 19/05/2022 at 15.05 ... now I have done the update of the skin then I have the live update screen that starts regularly .. ..Everything works also at the level of graphics that are updated but I don't understand what doesn't work. you can help me? thank you

![Screenshot_2022-05-20-05-13-36-022_com.android.chrome.jpg](https://user-images.githubusercontent.com/72796407/169445409-1cac00aa-e3f3-4074-903d-79a238204a6a.jpg)
![Screenshot_2022-05-20-05-13-43-453_com.android.chrome.jpg](https://user-images.githubusercontent.com/72796407/169445420-b425475d-07dc-4f91-9f8e-7b68bc4d6dea.jpg)

**Comment by poblabs** [comment, marked as answer]
Everything looks good to me when I view your website

**Reply by fvirgola80** [reply]
thanks Pat i updated the belchertown.js.tmpl file and now it works correctly .... i had the problem on mqtt thank you so much
